lib/app.rb

```diff
@@ -158,13 +174,16 @@ def redis_client
   tls_enabled = ENV['tls_enabled'] || false
 
   if redis_credentials.key?('sentinels')
-    @client ||= RedisClient.using_sentinel(redis_credentials)
+    if tls_enabled
+      @client ||= RedisClient.using_sentinel_tls(redis_credentials, version='TLSv1_2')
```

Comment:
(cosmetic) can we rename this method to be `tls_using_sentinel`

Reply:
Done